lib/Chat/Parser/SystemMessage.php

```diff
@@ -229,6 +272,11 @@ public function parseMessage(Message $chatMessage): void {
 				$parsedMessage = $this->l->t('You demoted {user} from moderator');
 			} elseif ($participant->isGuest() && $participant->getSessionId() === $parsedParameters['user']['id']) {
 				$parsedMessage = $this->l->t('{actor} demoted you from moderator');
+				if ($cliIsActor) {
+					$parsedMessage = $this->l->t('An administrator demoted you from moderator');
+				}
+			} elseif ($cliIsActor) {
+				$parsedMessage = $this->l->t('An administrator demoted {user} from moderator');
```

Comment:
`guest_moderator_demoted` is never reached since we can't demote guests from moderators via CLI.